src/org/folio/client/jira/JiraConstants.groovy

```diff
@@ -0,0 +1,16 @@
+package org.folio.client.jira
+
+import org.folio.client.jira.model.JiraResources
+
+class JiraConstants {
+
+  static final String URL = 'https://folio-org.atlassian.net'
```

Comment:
Not sure about this move from global vars to client's side...

Reply:
It will be rebased in the following PRs.